providers/class-two-factor-fido-u2f.php

```diff
@@ -251,23 +257,23 @@ public static function add_security_key( $user_id, $register ) {
 
 		if (
 			! is_object( $register )
-				|| ! property_exists( $register, 'keyHandle' ) || empty( $register->keyHandle )
-				|| ! property_exists( $register, 'publicKey' ) || empty( $register->publicKey )
+				|| ! property_exists( $register, 'key_handle' ) || empty( $register->key_handle )
```

Comment:
This would need to be reverted if we're keeping the vendor libraries away from WPCS.